gradle/libs.versions.toml

```diff
@@ -1,10 +1,10 @@
 [versions]
-android-plugin = "7.1.2"
+android-plugin = "8.0.0-alpha11"
```

Comment:
AGP version 7.3+ is required for Java 17 support

* v7.3 was failing complaining about incorrect path to generated sources (which were correct)
* v7.4 was failing due to https://issuetracker.google.com/issues/261329823